src/sentry/search/events/fields.py

```diff
@@ -1314,6 +1314,34 @@ def with_default(default, argument):
     return argument
 
 
+class SnQLNumericColumnNoLookup(NumericColumn):
+    measurement_aliases = {
+        MEASUREMENTS_FRAMES_SLOW_RATE,
+        MEASUREMENTS_FRAMES_FROZEN_RATE,
+        MEASUREMENTS_STALL_PERCENTAGE,
+    }
+
+    def __init__(self, name, allow_array_value=False):
+        super().__init__(name)
+        self.allow_array_value = allow_array_value
+
+    def normalize(self, value, params):
+        super().normalize(value, params)
+        return value
+
+    def get_type(self, value):
+        # `measurements.frames_frozen_rate` and `measurements.frames_slow_rate` are aliases
+        # to a percentage value, since they are expressions rather than columns, we special
+        # case them here
+        if isinstance(value, list):
```

Comment:
This is JSON specific syntax, I don't think you'd ever get value type as list here

Reply:
👍  removed this condition!